plugins/ltac/tacinterp.ml

```diff
@@ -687,7 +687,7 @@ let interp_may_eval f ist env sigma = function
   | ConstrContext ({loc;v=s},c) ->
       (try
 	let (sigma,ic) = f ist env sigma c in
-	let ctxt = coerce_to_constr_context (Id.Map.find s ist.lfun) in
+        let ctxt = try_interp_ltac_var coerce_to_constr_context ist (Some (env, sigma)) (make ?loc s) in
```

Comment:
At first view, an eta-contraction of the `{loc;v=s}` is possible.

Otherwise assigning.

Reply:
Pedrot, are you ok for this mini-simplification (I can then do it at the time of merging if you want)?

Reply:
@herbelin If you eta-contract `{loc;v=s}`, how should `user_err ?loc ~hdr:"interp_may_eval" (str "Unbound context identifier" ++ Id.print s ++ str".")` be updated?

Reply:
We still need `loc` for the `?loc` argument to `user_err` anyway, right?